examples/mugen/retrieval/video_clip.py

```diff
@@ -155,13 +169,15 @@ def forward(self, x: torch.Tensor) -> torch.Tensor:
         return embeds
 
 
-def videoclip(
+def build_videoclip(
```

Comment:
Sorry my bad. I checked again that we didn't use `build` prefix in other builders. 

https://github.com/facebookresearch/multimodal/blob/main/torchmultimodal/models/clip.py